# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: modules/libpref/init/StaticPrefList.yaml
# Commit: 7bacec3b6ce1
# Full Hash: 7bacec3b6ce17faf630b808ffbac94ce6f125c78
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2022-08-17 15:40:28
# Regressor Bug: 1740530
# File Overlap Count: 1
# Description:
#   Bug 1740530 - patch 5 - Implement support for COLRv1 glyphs represented as acyclic graphs of paint records. r=gfx-reviewers,lsalzman
#   
#   Depends on D153870
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D152041
# ==============================================================================

diff -r 4354a0d7248f -r 7bacec3b6ce1 modules/libpref/init/StaticPrefList.yaml
--- a/modules/libpref/init/StaticPrefList.yaml	Wed Aug 17 10:59:34 2022 +0000
+++ b/modules/libpref/init/StaticPrefList.yaml	Wed Aug 17 10:59:35 2022 +0000
@@ -5751,6 +5751,11 @@
     mirror: always
 #endif
 
+- name: gfx.font_rendering.colr_v1.enabled
+  type: RelaxedAtomicBool
+  value: false
+  mirror: always
+
 - name: gfx.font_rendering.opentype_svg.enabled
   type: RelaxedAtomicBool
   value: true
